src/explore-education-statistics-common/src/components/ErrorSummary.tsx

```diff
@@ -14,6 +15,7 @@ export interface ErrorSummaryMessage {
 
 interface BaseErrorSummaryProps {
   children: ReactNode;
+  screenReaderOnly?: boolean;
```

Comment:
Fairly fine as-is, but it's a little verbose so might be tempted by other options:

- `srOnly` (shorter, pretty common in various other libraries)
- `visuallyHidden` (more closely aligns with the name of the style)

Reply:
I've gone for visuallyHidden